examples/avatar/README.md

```diff
@@ -24,7 +24,7 @@ go build
 ### Usage
 
 This example uses bot tokens for authentication only. While email/password is 
-supported by DiscordGo, it is not recommended to use them.
+supported by DiscordGo, their use is not recommended.
```

Comment:
DiscordGo dropped all user account endpoints, I think would be better to remove this line.